docs/OV_Runtime_UG/supported_plugins/CPU.md

```diff
@@ -315,7 +321,7 @@ Read-only properties
 External Dependencies
 ###########################################################
 For some performance-critical DL operations, the CPU plugin uses optimized implementations from the oneAPI Deep Neural Network Library 
-(`oneDNN <https://github.com/oneapi-src/oneDNN>`__).
+(`oneDNN <https://github.com/oneapi-src/oneDNN>`__) and (`Compute Library <https://github.com/ARM-software/ComputeLibrary>`__).
```

Comment:
Whould rewtire a little bit: `For some performance-critical DL operations, the CPU plugin uses thirdaprty libraries`: 
- OneDNN (Intel, Arm) + link :
- ACL (Arm) + link.

Lists of operations I would propose to delete at all.

Reply:
Fixed.